src/Featurizers/Components/DocumentStatisticsEstimator.h

```diff
@@ -208,27 +256,32 @@ using DocumentStatisticsEstimator           = Components::TrainingOnlyEstimatorI
 
 // ----------------------------------------------------------------------
 // |
-// |  DocumentStatisticsAnnotationData
+// |  FrequencyAndIndexStruct
 // |
 // ----------------------------------------------------------------------
-inline DocumentStatisticsAnnotationData::DocumentStatisticsAnnotationData(FrequencyMap termFreq, IndexMap termIndex, std::uint32_t totalNumDocuments) :
-    TermFrequency(
-        std::move(
-            [&termFreq](void) -> FrequencyMap & {
-                if(termFreq.empty())
-                    throw std::invalid_argument("termFreq");
+FrequencyAndIndexStruct::FrequencyAndIndexStruct(std::uint32_t termFrequency, std::uint32_t index) :
+    //validate not necessary in this assignment
+    TermFrequency(std::move(termFrequency)),
```

Comment:
>TermFrequency(std::move(termFrequency)), [](start = 4, length = 40)

Is it valid for a termFrequency to be 0?

Reply:
it won't happen, will add validation there